CONTRIBUTING.md

```diff
@@ -11,7 +11,36 @@ No software is bug free. So, if you got an issue, follow these steps:
   * Clear title (make is shorter if possible).
   * Describe the issue in clear language.
   * Share error logs, screenshots and etc.
-  * To speed up the issue fixing process, send us a sample repo with the issue you faced.
+  * To speed up the issue fixing process, send us a sample repo with the issue you faced:
+  
+### Reproductions
+
+The best way to help figure out an issue you are having is to produce a minimal reproduction. A good way to do that is using Create React App:
+
+```bash
+# if necessary
+npm install -g create-react-app getstorybook
+
+create-react-app reproduction
+cd reproduction
+getstorybook
+
+# make changes to try and reproduce the problem, such as adding components + stories
+yarn start storybook
```

Comment:
`getstorybook` outputs the above `yarn` command so it would be a bit confusing to then tell people to use `npm` here. Maybe I'll change it to `yarn global add ...`?